src/PKSim.Infrastructure/Services/IndividualPropertiesCacheImporter.cs

```diff
@@ -39,7 +39,7 @@ public IndividualPropertiesCache ImportFrom(string fileFullPath, IImportLogger l
                var individualPropertyCache = individualPropertiesCacheFrom(fileFullPath, delimiter);
                //we found at least one individual, this is a valid file for the delimiter and we can exit
                if (individualPropertyCache?.Count > 0)
-                  return individualPropertyCache;
+                  return withPathsContainingUnitsUpdated(individualPropertyCache, allParameters);
```

Comment:
making sure that we update the path of parameter containing units before returning int That way, anyone using this object will have clean values